xla/stream_executor/device_description.h

```diff
@@ -69,7 +69,7 @@ struct CudaComputeCapability {
   }
 
   bool Is(int other_major, int other_minor = 0) const {
-    return !(*this == CudaComputeCapability{other_major, other_minor});
+    return *this == CudaComputeCapability{other_major, other_minor};
   }
```

Comment:
This function checks both the major and minor version. I see at https://developer.nvidia.com/cuda-gpus that some Ampere cards have compute capability 8.6 and so the check of `cuda_compute_capability.Is(se::CudaComputeCapability::AMPERE)` will not pass for these cards. I would remove this function and just check the major version in cudnn_norm_rewriter.cc.